catalog/app/containers/Bucket/PackageCreateDialog.js

```diff
@@ -473,9 +473,7 @@ function PackageCreateDialog({ bucket, open, workflowsConfig, onClose, refresh }
           scroll="body"
           onExited={reset(form)}
         >
-          <M.DialogTitle>
-            {success ? 'Package created' : 'Create package'}
-          </M.DialogTitle>
+          <M.DialogTitle>{success ? 'Package created' : 'Create package'}</M.DialogTitle>
```

Comment:
I don't understand why my ESLint skipped this

Reply:
maybe bc it was a suggestion merged via github web ui